apps/dashboard/src/contract-ui/tabs/proposals/components/proposal-button.tsx

```diff
@@ -15,55 +26,93 @@ interface VoteButtonProps {
 const PROPOSAL_FORM_ID = "proposal-form-id";
 
 export const ProposalButton: React.FC<VoteButtonProps> = ({ contract }) => {
-  const { isOpen, onOpen, onClose } = useDisclosure();
-
+  const [open, setOpen] = useState(false);
   const sendTx = useSendAndConfirmTransaction();
-
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<{ description: string }>();
+  const trackEvent = useTrack();
   return (
-    <>
-      <Drawer
-        size="md"
-        isOpen={isOpen}
-        onClose={onClose}
-        header={{ children: <>Create new proposal</> }}
-        footer={{
-          children: (
-            <>
-              <Button
-                isDisabled={sendTx.isPending}
-                variant="outline"
-                mr={3}
-                onClick={onClose}
-              >
-                Cancel
-              </Button>
-              <TransactionButton
-                txChainID={contract.chain.id}
-                transactionCount={1}
-                isLoading={sendTx.isPending}
-                form={PROPOSAL_FORM_ID}
-                type="submit"
-                colorScheme="primary"
-              >
-                Submit Proposal
-              </TransactionButton>
-            </>
-          ),
-        }}
-      >
-        <CreateProposalForm
-          formId={PROPOSAL_FORM_ID}
-          contract={contract}
-          sendTx={sendTx}
-        />
-      </Drawer>
-      <Button
-        colorScheme="primary"
-        onClick={onOpen}
-        leftIcon={<PlusIcon className="size-5" />}
-      >
-        Create Proposal
-      </Button>
-    </>
+    <Sheet open={open} onOpenChange={setOpen}>
+      <SheetTrigger asChild>
+        <Button
+          colorScheme="primary"
+          leftIcon={<PlusIcon className="size-5" />}
+        >
+          Create Proposal
+        </Button>
+      </SheetTrigger>
+      <SheetContent className="z-[10000] sm:w-[540px] sm:max-w-[90%] lg:w-[700px]">
+        <SheetHeader>
```

Comment:
Lets increase the max-width on mobile ( < sm ) , there is not enough space to fit cancel + button text

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/7ad2921e-ceca-4083-9289-55d6267e3bd7.png)

Reply:
Updated and fixed the UIs in other places as well. thanks ^